linux/shim/init.go

```diff
@@ -347,3 +348,13 @@ func copyFile(to, from string) error {
 	_, err = io.Copy(tt, ff)
 	return err
 }
+
+func checkKillError(err error) error {
+	if err == nil {
+		return nil
+	}
+	if strings.Contains(err.Error(), "os: process already finished") {
```

Comment:
I'm not sure i can count on the concrete type because its being return by runc where the error is happening and not from a syscall.